tools/metricsdocs/metricsdocs.go

```diff
@@ -3,10 +3,11 @@ package main
 import (
```

Comment:
this file should move under pkg/monitoring/tools

Reply:
Tools should be in the pkg/monitoring! Many component's Dockefiles when building the operator copy pkg/ into the final image, and it is not necessary for the final build, so it should be in a separate base directory like just tools/

Reply:
In cdi we already moved the metrics tools under monitoring, and also I was able to move them in hco and hpp.
Basically it was Shirly's request so adding here @sradco